.github/workflows/release.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
           yarn install
           yarn tsc
           yarn build
-          if [ $(npm publish --dry-run 2>&1 | grep dist) ]
+          if HAS_ARTIFACTS=$(npm publish --dry-run 2>&1 | grep dist)
```

Comment:
This was stolen from https://unix.stackexchange.com/questions/430017/redirect-command-error-to-variable-in-if-statement